Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -130,6 +123,22 @@ public class MXRAndroidSystem : IMXRSystem {
             };
         }
 
+        HashSet<string> executedIntentIds = new HashSet<string>();
+        void TryExecuteIntentCommands() {
+            var intentId = MXRAndroidUtils.GetExtraString("intentId");
```

Comment:
A generic key based function makes sense to me, but the upside of using the existing function which gets intentId specifically is that you can't make a typo when reusing it, and we're limiting ourselves very clear to a known set of extra strings.

Up to you, just wanted to make the suggestion!